app/mailers/user_mailer.rb

```diff
@@ -4,106 +4,164 @@ class UserMailer < ActionMailer::Base
 
   def password_reset_instructions(user)
     @user = user
-    mail(subject: 'openSNP.org Password Reset Instructions', to: user.email)
+    mail(subject: 'openSNP.org Password Reset Instructions', to: user.email) do |format|
+      format.html { render layout: 'user_mailer.html.erb' }
+      format.text { render layout: 'user_mailer.text.erb' }
+    end
```

Comment:
You can set the layout for the whole mailer, instead of having to do it over and over again. This way you don"t even need the format block: http://guides.rubyonrails.org/v4.2/action_mailer_basics.html#action-mailer-layouts